src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -356,14 +359,52 @@ const Hole::ThreadDescription Hole::threadDescription[][171] =
 
 };
 
+const double Hole::metricHoleDimensions[32][4] =
+{
+    /* ISO metric according to ISO 273 */
+    // {screw diameter, close fit, standard fit, coarse fit}
+        { 1.0,      1.7,    1.8,    2.0},

```

Comment:
This seems suspicious: the book I've got lists 1.05, 1.1, and 1.2, respectively, for the M1.

Reply:
Thanks, copy/paste mistake by me. I will fix this.